.upgrade.yml

```diff
@@ -953,3 +953,151 @@ excludedPaths:
   - '*vendor*'
 doctorTasks:
   SilverStripe\Dev\Upgrade\UpgradeBootstrap: src/Dev/Upgrade/UpgradeBootstrap.php
+warnings:
+  classes:
+    'Object':
+      message: 'Replaced with traits'
+      url: 'https://docs.silverstripe.org/en/4/changelogs/4.0.0#object-replace'
+    'SS_Log':
+      message: 'Replaced with a PSR-3 logger'
+      url: 'https://docs.silverstripe.org/en/4/changelogs/4.0.0#psr3-logging'
+    'Session':
+      message: 'Session is no longer statically accessible'
+      url: 'https://docs.silverstripe.org/en/4/changelogs/4.0.0#session'
+    'SQLQuery':
+      message: 'Use SQLSelect instead'
+      url: 'https://docs.silverstripe.org/en/4/changelogs/4.0.0#sqlquery'
+    'SilverStripe\AssetAdmin\Forms\UploadField':
+      message: 'Class implementations should be wrapped with FileHandleField'
```

Comment:
I don't agree this is necessary; Direct references to the new UploadField::create() are fine.